runtime/litentry/src/xcm_config.rs

```diff
@@ -299,13 +300,7 @@ parameter_types! {
 
 impl pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	// We allow anyone to send any XCM to anywhere
-	// This is highly relied on if target chain properly filtered
-	// Check their Barriers implementation
-	// And for TakeWeightCredit
-	// Check if their executor's ShouldExecute trait weight_credit
-	type SendXcmOrigin =
-		FilterEnsureOrigin<RuntimeOrigin, LocalOriginToLocation, EnsureRootOrTwoThirdsCouncil>;
+	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
```

Comment:
Since we do not have orml_xcm any more. Be aware that this impl will make us not able to make sovereign account do anything (even root).